From 9b2cb33a3d8c5389a057f905a50c5ac3a498d69c Mon Sep 17 00:00:00 2001
From: Doug Ledford <dledford@redhat.com>
Date: Wed, 24 Sep 2014 14:58:22 -0400
Subject: [infiniband] usnic: Advertise usNIC devices as RDMA_NODE_USNIC_UDP

Message-id: <0413844d0141a5963d7a42e8920b8c9032dcda85.1411569661.git.dledford@redhat.com>
Patchwork-id: 96585
O-Subject: [Patch RHEL7 20/29] IB/usnic: Advertise usNIC devices as RDMA_NODE_USNIC_UDP
Bugzilla: 916382
RH-Acked-by: Jes Sorensen <Jes.Sorensen@redhat.com>

From: Upinder Malhi <umalhi@cisco.com>

usNIC default transport is UDP.  Hence, advertise RDMA_NODE_USNIC_UDP
by default for usNIC devices.

Signed-off-by: Upinder Malhi <umalhi@cisco.com>
Signed-off-by: Roland Dreier <roland@purestorage.com>
(cherry picked from commit 61f78268936e781a104b4ac06b7e47d760800c40)
Signed-off-by: Doug Ledford <dledford@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/infiniband/hw/usnic/usnic_ib_main.c b/drivers/infiniband/hw/usnic/usnic_ib_main.c
index 7d2efd4..fb6d026 100644
--- a/drivers/infiniband/hw/usnic/usnic_ib_main.c
+++ b/drivers/infiniband/hw/usnic/usnic_ib_main.c
@@ -332,7 +332,7 @@ static void *usnic_ib_device_add(struct pci_dev *dev)
  us_ibdev->pdev = dev;
  us_ibdev->netdev = pci_get_drvdata(dev);
  us_ibdev->ib_dev.owner = THIS_MODULE;
- us_ibdev->ib_dev.node_type = RDMA_NODE_USNIC;
+ us_ibdev->ib_dev.node_type = RDMA_NODE_USNIC_UDP;
  us_ibdev->ib_dev.phys_port_cnt = USNIC_IB_PORT_CNT;
  us_ibdev->ib_dev.num_comp_vectors = USNIC_IB_NUM_COMP_VECTORS;
  us_ibdev->ib_dev.dma_device = &dev->dev;
-- 
1.7.1